manifests/params.pp

```diff
@@ -17,7 +17,12 @@
   $puppetdb_version          = $puppetdb::globals::version
   $database                  = $puppetdb::globals::database
   $manage_dbserver           = true
-  $manage_pg_repo            = true
+
+  if $::osfamily =~ /RedHat|Debian/ {
```

Comment:
@ekohl You're right, that does to the same thing. 

Unless @mullr requests it for this PR, though, I'm going to keep this PR as is. We (or Puppet) could always submit that change as a part of a PR to bring the module in-line with Puppet 4.